backend/workflow_manager/endpoint/constants.py

```diff
@@ -9,6 +9,15 @@ class DBConnectionClass:
     BIGQUERY = "Client"
 
 
+class UnstractDBConnectorClass:
```

Comment:
If we adding a class in connectors, Are we need to add here also?

Reply:
We need this. Let create-sql query be on `database_utils`.
create-sql query different for different connector. So lets keep it here